client/controller/models/certs/certs_test.go

```diff
@@ -271,6 +275,15 @@ func TestCertAttach(t *testing.T) {
 	if err = Attach(&client, "test-example-com", "foo.com"); err != nil {
 		t.Fatal(err)
 	}
+
+	if err = Attach(&client, "non-existent-cert", "foo.com"); err == nil {
+		t.Fatal("An Error should have resulted from the attempt to attach a non-existent cert to a valid domain")
+	}
+
+	// TODO: enable once Attach can determine if domain is valid
```

Comment:
@vdice could you add an issue on this repo for making `Attach` determine if a domain is valid (and expanding these tests to match)? I'm good with this comment & commented out code also, if that issue exists